servers/jax-rs-tests/build.gradle.kts

```diff
@@ -53,6 +53,7 @@ dependencies {
   testImplementation(project(":nessie-versioned-storage-jdbc"))
 
   testImplementation(project(":nessie-jaxrs-testextension"))
+  testImplementation(libs.hibernate.validator.cdi)
```

Comment:
Required, otherwise none of the `@NotNull` annotations and alike are actually enforced in this module.